build/toolchain-aarch64-linux-android.cmake

```diff
@@ -31,10 +31,13 @@ set(DIST_ARCH arm64-v8a)
 string(TOLOWER ${CMAKE_HOST_SYSTEM_NAME} HOST_NAME_L)
 file(TO_CMAKE_PATH $ENV{ANDROID_HOME} ANDROID_HOME_UNIX)
 
-file(GLOB NDK_VERSIONS LIST_DIRECTORIES true ${ANDROID_HOME_UNIX}/ndk/*)
-list(SORT NDK_VERSIONS)
-list(GET NDK_VERSIONS -1 NDK_VERSION)
-get_filename_component(NDK_VERSION ${NDK_VERSION} NAME)
+set(NDK_VERSION $ENV{FILAMENT_NDK_VERSION})
```

Comment:
So here I'd rather we matched the major version number and pick for instance the newest NDK 22 installed. We could also print a warning if the major version matches but not the rest of the version number.

Reply:
Should that be done in a CMake part or in `build.sh`, assuming CMake was not planned to be standalone as far as I've understood?

Reply:
Added some changes to address that.

Reply:
`build.sh` is just a helper, the CMake files can be used standalone.